src/js/components/ServiceDetail.js

```diff
@@ -30,7 +37,10 @@ class ServiceDetail extends mixin(InternalStorageMixin, TabsMixin) {
 
     this.state = {
       currentTab: Object.keys(this.tabs_tabs).shift(),
-      isServiceFormModalShown: false
+      isServiceFormModalShown: false,
+      isServiceDestroyConfirmShown: false,
+      isServiceScaleFormModalShown: false,
+      isServiceSuspendConfirmShown: false
```

Comment:
Good idea, at least this looks much better then. Underlaying problem will remain the same..
Will do it.